Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/Get.ts

```diff
@@ -10,7 +10,10 @@ import { findDocumentByIdSql } from './SqlHelper';
 
 const moduleName = 'postgresql.repository.Get';
 
-export async function getDocumentById({ documentUuid, traceId }: GetRequest, client: PoolClient): Promise<GetResult> {
+export async function getDocumentByDocumentUuid(
+  { documentUuid, traceId }: GetRequest,
+  client: PoolClient,
+): Promise<GetResult> {
   // TODO *** cast to unknown is an invalid mixing of meadowlarkId and documentUuid
   const meadowlarkId: MeadowlarkId = documentUuid as unknown as MeadowlarkId;
   try {
```

Comment:
Line 21, shouldn't that be calling `findDocumentByDocumentUuidSql` instead of `findDocumentByIdSql`?

Reply:
This makes me think that we're missing some test automation. @andonyns can you review and offer suggestion for where we can update the e2e tests to make sure that there is a GET BY ID test that fails with the current code, but passes after the above update is made.

Reply:
I will take a look